src/webapp/features/rabble/GameScreen.tsx

```diff
@@ -34,7 +34,9 @@ const GameScreen = (props: GameBoardProps) => {
   return (
     <div className={styles.board}>
       <div className={styles.topButtonContainer}>
-        <InviteLink gameID={gameID} />
+        {scoreList[1] && !scoreList[1].nickname.length ? (
```

Comment:
Can we also add an `Array.isArray(scoreList)` check here?  it'll prevent a crash if scoreList is for some reason undefined.

I bet we could use optional chaining too:
```
Array.isArray(scoreList) && scoreList[1]?.nickname?.length ? null : <InviteLink>
```

Or something similar

Reply:
got it !